.github/workflows/kind_e2e.yaml

```diff
@@ -121,7 +121,18 @@ jobs:
     - name: Free up some diskspace
       run: |
         docker image prune -a -f
-
+    - name: Determine latest Dapr Runtime version including Pre-releases
```

Comment:
@mukundansundar Would this make builds not reproducible for release branches going forward? I remember pinning the runtime version in our Java SDK IT tests to make sure old releases continue to build the same way.

Reply:
I would keep the other changes but pin the runtime and dashboard versions here instead.

Reply:
Another option is to combine both where if in master branch use the latest but if from a release branch, use a pinned version.

Reply:
I agree @artursouza. Pinning the versions in the release branch and using the latest versions in the master branch and associated PRs makes perfect sense. 
@amulyavarote can you make changes such that PRs targeting release branch and the release branch itself have pinned versions whereas the master branch and PRs targeting master branch have the latest versions?